### ISSUE TITLE
Breaking builds

### ISSUE THREAD

**Issue by bkniffler** [szmarczak/http2-wrapper#63]
Latest version is breaking builds. I've installed got, not using http2-wrapper directly, and nodejs15 is required.

**Comment by tabrez96** [comment]
Facing the same issue.

**Comment by cesine** [comment]
Same for us, @szmarczak can you relax the versions for a few hours? i will prep a pr



```
10:27:51  error http2-wrapper@1.0.0: The engine "node" is incompatible with this module. Expected version ">=15.10.0". Got "12.13.1"

```

**Comment by pataar** [comment]
Same here

**Comment by domtaylor** [comment]
Same here! Hope you can fix asap

**Comment by gustavkj** [comment]
I have the same issue. But a solution seems to be in the making over on `got`: https://github.com/sindresorhus/got/pull/1640

**Comment by szmarczak** [comment]
Fixed. I thought that the `^` would up the beta version into a stable one.

**Comment by szmarczak** [comment]
😅

**Comment by szmarczak** [comment]
Ok so I released `1.0.3` which is 100% the same beta version. The new version is released as `2.0.0`. Sorry for the trouble.
